### ISSUE TITLE
[Bug] 训练的时候报错，标签问题？我设置的2我的样本标签也只有2为什么还报错

### ISSUE THREAD

**Issue by xinyujituan** [PaddlePaddle/PaddleSeg#2525]
D:\ProgramData\anaconda3\envs\Paddle\python.exe "D:\Program Files\JetBrains\PyCharm 2020.1\plugins\python\helpers\pydev\pydevd.py" --multiproc --qt-support=auto --client 127.0.0.1 --port 50550 --file D:/Python/PaddleSeg-release-2.6/train.py --config configs/quick_start/deeplabv3p_resnet18_os8_optic_disc_512x512_1k_student.yml --do_eval --use_vdl --save_interval 500 --save_dir output
pydev debugger: process 9348 is connecting
Connected to pydev debugger (build 201.6668.115)
2022-09-09 16:03:17 [INFO]	
------------Environment Information-------------
platform: Windows-10-10.0.19041-SP0
Python: 3.7.11 (default, Jul 27 2021, 09:42:29) [MSC v.1916 64 bit (AMD64)]
Paddle compiled with cuda: True
NVCC: Build cuda_11.2.r11.2/compiler.29373293_0
cudnn: 8.1
GPUs used: 1
CUDA_VISIBLE_DEVICES: None
GPU: ['GPU 0: NVIDIA GeForce']
PaddleSeg: 2.6.0
PaddlePaddle: 2.3.1
OpenCV: 4.2.0
------------------------------------------------
2022-09-09 16:03:17 [INFO]	
---------------Config Information---------------
batch_size: 4
distill_loss:
  coef:
  - 3
  types:
  - type: KLLoss
iters: 1000
loss:
  coef:
  - 1
  types:
  - ignore_index: 255
    type: CrossEntropyLoss
lr_scheduler:
  end_lr: 0
  learning_rate: 0.01
  power: 0.9
  type: PolynomialDecay
model:
  align_corners: false
  aspp_out_channels: 256
  aspp_ratios:
  - 1
  - 12
  - 24
  - 36
  backbone:
    multi_grid:
    - 1
    - 2
    - 4
    output_stride: 8
    pretrained: null
    type: ResNet18_vd
  backbone_indices:
  - 0
  - 3
  num_classes: 2
  pretrained: null
  type: DeepLabV3P
optimizer:
  momentum: 0.9
  type: sgd
  weight_decay: 4.0e-05
train_dataset:
  dataset_root: data/output
  mode: train
  transforms:
  - target_size:
    - 512
    - 512
    type: Resize
  - type: RandomHorizontalFlip
  - type: Normalize
  type: OpticDiscSeg
val_dataset:
  dataset_root: data/output
  mode: val
  transforms:
  - target_size:
    - 512
    - 512
    type: Resize
  - type: Normalize
  type: OpticDiscSeg
------------------------------------------------
W0909 16:03:20.197811  4764 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 11.6, Runtime API Version: 11.2
W0909 16:03:20.197811  4764 gpu_resources.cc:91] device: 0, cuDNN Version: 8.1.
2022-09-09 16:03:21 [INFO]	No pretrained model to load, ResNet_vd will be trained from scratch.
W0909 16:03:21.275944  4764 gpu_resources.cc:201] WARNING: device:

**Comment by shiyutang** [comment]
根据提供的内容没有看到你的报错信息

**Comment by juncaipeng** [comment]
标注的样本标签必须从0开始增加，如果有两类，必须是0和1，不可以是0和2
